### PR TITLE
[WIP]: Feature/GitHub permissions

### PR DESCRIPTION
**Pull request by cotts**
Working on related tasks #21, #31 

- [x] Get information given organization with tests
- [x] Refactor code to use Class Based on Github lib 
- [x] Testing script to store data on boot the app 
- [x] Create Script to store data before app boot
- [x] Authorization methods with tests
- [x] Update Readme with Env and requirements

I had to refactor the structure created before to use the stored data instead of making requests to GitHub API

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           browser: chrome
-          build: yarn run build
-          start: yarn start
+          build: next build
+          start: npm start
```
Comment (risenW) [inline .github/workflows/main.yml:20]: Hey cotts, just to point at that we use yarn as the default package manager in most of our project

**Review by risenW — CHANGES_REQUESTED** [review]

**Comment by risenW** [comment]
@cotts I added a comment and you might want to check why the tests are failing

File: .github/workflows/main.yml (hunk 1 above)
Comment (cotts) [reply .github/workflows/main.yml:20]: ok, I will change that.

Comment (cotts) [reply .github/workflows/main.yml]: done

**Review by risenW — APPROVED** [review]

**Comment by risenW** [comment]
@cotts Looks good to merge

**Pull request merged**